### ISSUE TITLE
use of non standard library module fails

### ISSUE THREAD

**Issue by Hades32** [AtomLinter/linter-elixirc#12]
For example when I create a new Phoenix project with a channel, there is something like
``` Elixir
# ./web/channels/game_channel.ex
defmodule Square.GameChannel do
  use Square.Web, :channel
```
and this
```Elixir
# ./web/web.ex
defmodule Square.Web do
# ...
defmacro __using__(which) when is_atom(which) do
# ...
```

The line with the "use" is marked as erronous with "module Square.Web is not loaded and could not be found".

When compiling and executing, everything works fine, though.

Maybe related to https://github.com/AtomLinter/linter-elixirc/issues/10

**Comment by remiq** [comment]
@Hades32: Please check latest version of the plugin - 1.0.0
If `mix.exs` exists in project's directory, plugin will use `mix compile`.

**Comment by Hades32** [comment]
Looks good! Thanks!
